sphinx/templates/quickstart/conf.py_t

```diff
@@ -112,12 +112,16 @@ html_static_path = ['{{ dot }}static']
 #
 # This is required for the alabaster theme
 # refs: http://alabaster.readthedocs.io/en/latest/installation.html#sidebars
-html_sidebars = {
-    '**': [
-        'relations.html',  # needs 'show_related': True theme option to display
-        'searchbox.html',
-    ]
-}
+if html_theme == 'alabaster':
+    html_sidebars = {
+        '**': [
+            'about.html',
+            'navigation.html',
+            'relations.html',  # needs 'show_related': True theme option to display
+            'searchbox.html',
+            'donate.html',
```

Comment:
not clear to me if donate should be included per default

Reply:
This was what it was before being removed in https://github.com/sphinx-doc/sphinx/pull/4002. [donate.html](https://github.com/bitprophet/alabaster/blob/da3b2595b95bc94deacc45c617096ea57ce6790a/alabaster/donate.html) is activated conditionally by Alabaster from additional theme configuration. It won't be shown by default.

This is the same default we will get with [#4177 - html.py](https://github.com/sphinx-doc/sphinx/pull/4177/files#diff-c17608ed63bfe1714c6befb9d4201feeR878), and probably will be the same once `sidebars` is added to Alabaster's `theme.conf`. Of course, that is for 1.7 (master). This PR is for 1.6 (stable).

Reply:
ok no strong opinion on this